xml/docu_styleguide.structure.xml

```diff
@@ -1516,8 +1516,8 @@ xml:id="tab-install-source"</screen>
    <listitem>
     <para>
      If all list items are nouns only, do not capitalize their first letter.
-     Use sentence-style capitalization for list items that are full sentences.
-     Use title-style capitalization for terms in descriptive lists.
+     Use sentence-style capitalization for list items that are full sentences
```

Comment:
```suggestion
     Use sentence-style capitalization for list items that are full sentences,
```

Reply:
Liam, when you make suggestions like this, please explain them. :/ This is a listing of two items. Why is there a need for a comma?